docs/administration/security/hardening-octopus.md

```diff
@@ -253,6 +253,23 @@ The TCP ports listed below are defaults, and can be changed if required - refer
 |WinRM-HTTP|`TCP 5985`|Octopus Server|Anywhere|DENY|Prevents attackers from using the Octopus Server as a beachhead into your network via unsecured WinRM.|
 |All outbound|`ALL`|Octopus Server|Anywhere|???|Depends on how fine-grained control you want over what your Octopus Server can do. It also depends on where your workers are, and where you are deploying to. Allowing all outbound traffic is a good place to start, then perform network analysis to decide on your next step.|
 
+## Harden your containers
+
+If you run an [Octopus Deploy container](/docs/installation/octopus-in-container/index.md) these are a few of the steps you can take to secure it. 
```

Comment:
I'd make the language a bit more certain here:
```suggestion
If you run an [Octopus Deploy container](/docs/installation/octopus-in-container/index.md), you need to take the following steps to secure it: 
```
If we're worried about giving users the impression these are the only steps they need to take, we can `in addition to your usual security measure for running apps out of containers, you need to...